test/dependency-order.bats

```diff
@@ -34,3 +34,66 @@ EOF
     bad_stacker build
     echo "${output}" | grep "couldn't find dependencies for test: stacker://foo/bar, stacker://baz/foo"
 }
+
+@test "stacker:// style nesting w/ type built works" {
+    cat > stacker.yaml <<EOF
+minbase1:
+  from:
+    type: oci
+    url: $UBUNTU_OCI
+  run: |
+    echo pkgtool install cpio pigz
+
+installer-initrd:
+    build_only: true
+    from:
+        type: built
+        tag: minbase1
+    run: |
+        #!/bin/bash -ex
+        mkdir /output
+        touch /output/installer-initrd-base.cpio
+
+        echo "${STACKER_LAYER_NAME}" > /layer.txt
```

Comment:
you probably need to escape this if you want it to work.
I would expect that the STACKER_LAYER_NAME variable is rendered at time it is read.
It looks like you *want* that to happen as `$UBUNTU_OCI` probably needs to be rendered from the current environment.

Reply:
I think we can just drop this actually, it's copypasta from your minified example that's not critical to the test.